src/install/nginx/linux/install-tarball.mdx

```diff
@@ -0,0 +1,32 @@
+---
+componentType: default
```

Comment:
@bradleycamacho there seems to be a bug in the tarball logic. When I select this, I also see the generic section "Download the NGINX integration" which isn't necessary.